app/helpers/script_helper.rb

```diff
@@ -34,6 +35,19 @@ def local_crossorigin_sources?
     Rails.env.development? && ENV['WEBPACK_PORT'].present?
   end
 
+  def javascript_assets_tag(*names)
+    assets = AssetSources.get_assets(*names)
+    if assets.present?
+      asset_map = assets.index_with { |path| asset_path(path) }
```

Comment:
I expect I'll need to revisit this in future work, where we'll want to use this for the design system icon sprite, which can't be loaded across domains, so we'll need to have a way to pass `host: asset_host` like we do in `IconComponent`:

https://github.com/18F/identity-idp/blob/5a8af56afa7a88cf208ff130145585d81fe6c136/app/components/icon_component.rb#L261

Thinking we might need to have some sort of allowlist, e.g.

```suggestion
      asset_map = assets.index_with do |path|
        asset_path(path, host: SAME_ORIGIN_ASSETS.include?(path) && asset_host)
      end
```

(`host` option uses [falsey assignment for defaulting](https://github.com/rails/rails/blob/de53ba56cab69fb9707785a397a59ac4aaee9d6f/actionview/lib/action_view/helpers/asset_url_helper.rb#L279))